Feature/initial zoom

Morta1 opened the pull request.

danielcaldas commented on src/components/graph/graph.config.js:
```diff
@@ -229,6 +230,7 @@ export default {
     linkHighlightBehavior: false,
     maxZoom: 8,
     minZoom: 0.1,
+    initialZoom: 1,
```
Just curious here, why should the default value be `1` instead of `0`? I think this value should be `null` by default.

danielcaldas commented on src/components/graph/Graph.jsx:
```diff
@@ -277,13 +277,16 @@ export default class Graph extends React.Component {
+            .on("zoom", this._zoomed);
+
+        // set initial zoom
+        zoomObject.scaleTo(selector, this.state.config.initialZoom);
```
We should run this optionally based on the value provided by the client. If there's no value defined in `this.state.config.initialZoom` then no need to run this code.

```javascript
if (this.state.config.initialZoom !== null) {
    zoomObject.scaleTo(selector, this.state.config.initialZoom);
}
```

danielcaldas commented on src/components/graph/graph.config.js:
```diff
@@ -80,6 +80,7 @@
  * the value the more the less highlighted nodes will be visible (related to <i>nodeHighlightBehavior</i>).
  * @param {number} [maxZoom=8] - max zoom that can be performed against the graph.
  * @param {number} [minZoom=0.1] - min zoom that can be performed against the graph.
+ * @param {number} [initialZoom=1] - initial zoom that can be performed against the graph.
```
```suggestion
 * @param {number} [initialZoom=1] - initial zoom value to be set on the graph.
```

danielcaldas commented:
Hey, @Morta1 I think this feature is very useful, perhaps you could add a bit more of context/motivation for you to purpose this change?

antoninklopp commented:
@Morta1 any update on this?

StevenHeven commented:
Pleaze resolve conflict ! It will be very helpful ! Can you notice me when it merge please ? 
I've got some bug with focusZoom in confing not work and i think initial zoom will fix that ! :P

Choidaeboem commented:
This PR is a feature I need.

Morta1 commented:
@danielcaldas I've updated the PR.
The PR was originally created to increase the initial zoom of the graph.
The initial zoom of the graph was too far for my needs so I've added this small adjustment.
